variables.tf

```diff
@@ -154,6 +154,21 @@ variable "rhcos_kernel_options" {
     default     = []
 }
 
+variable "sysctl_tuned_options" {
+    description = "Set to true to apply sysctl options via tuned operator. Default: false"
+    default     = false
+}
+
+variable "sysctl_options" {
+    description = "List of sysctl options to apply."
+    default     = [""]
```

Comment:
This is list of 1 empty string.
Please make it []. We have example to show that strings are expected here,

Same comment for next variable.